### DISCUSSION TITLE
Move from Password Confirmation to Password Show/Hide

### DISCUSSION THREAD

**Discussion by mgifford** [uswds/uswds#5799] [Accessibility]
There is some history here https://github.com/uswds/uswds/issues/4592 which seems to indicate at one point there was a show/hide feature to login. 

There are a few good articles on why we should be doing this by default:
https://joshwayne.com/posts/show-password/
https://technology.blog.gov.uk/2021/04/19/simple-things-are-complicated-making-a-show-password-option/
https://design-patterns.service.justice.gov.uk/components/password-reveal/

Why is the default for resetting the password still this:
https://designsystem.digital.gov/templates/form-templates/password-reset-form/

**Comment by mejiaj** [comment]
@mgifford to confirm; you're saying we should flip the behavior of the show/hide password to **show** by default?

The first article recommends _against_ this. 

![image](https://github.com/uswds/uswds/assets/3385219/6884e02e-c719-413a-a341-e29748543788)

[Source →](https://joshwayne.com/posts/show-password/#:~:text=Show%20by%20default.%20Showing%20the%20password%20by%20default%20might%20seem%20like%20the%20most%20user%20friendly%20solution%2C%20but%20user%20testing%20shows%20that%20when%20people%20saw%20the%20password%20shown%20by%20default%2C%20they%20believed%20it%20was%20a%20bug%2C%20the%20site%20was%20hacked%2C%20or%20the%20site%20isn%E2%80%99t%20safe.)

**Comment by jaclinec** [comment]
@mgifford if you're suggesting we do away with the 'confirm password' and instead just allow users to enter the password once and show/hide password, then I agree. Although I can see how most folks probably WOULDN'T choose to show their password and if they mistype, they may be forced to go through a 'forgot password' flow down the line, but even so, I think the positives outweigh the potential negatives of getting rid of the 'confirm password.'
